libraries/.foodcritic

```diff
@@ -0,0 +1,4 @@
+~FC011
```

Comment:
I guess foodcritic is failing because we're running it from the wrong directory.

Actually, it should receive as parameter the `cookbooks/` folder, so it gets our `oneview` cookbook and then analyzes it. But in this case it is being executed inside `oneview/`, so this way it is interpreting the `libraries` as a separate cookbook, and not a subfolder